### DISCUSSION TITLE
Mobile support

### DISCUSSION THREAD

**Discussion by AmmarZeer** [xyflow/xyflow#2403] [Q&A]
I am working on a project that has a canvas component, I've been using react-flow to implement the canvas and so far I've managed to implement all the needed requirements for the component.

One thing am stuck at is mobile support, I know i can connect 2 nodes using a mobile by touching the handles, but does the library support other features for the mobile such as (implementing drag and drop for mobile, moving around the flow[canvas] with touch devices, etc..)

I've been trying to add onTouchMove and onTouchEnd as props to the <ReactFlow> component, but with no luck to make it mobile friendly.

Any advice on implementing this or to let me know if it's even possible to support mobile on this component.

**Comment by moklick** [comment]
All your mentioned features are supported by the library. An example for connecting nodes via click can be found here: https://reactflow.dev/docs/examples/interaction/touch-device/

**Reply by AmmarZeer** [reply]
@moklick Is there more documentation or examples for touch devices?
something for the onTouchMove or any touch events? as the component isn't calling the function like it does for onDragOver for example.
